docs/usage-tui.md

```diff
@@ -0,0 +1,207 @@
+# Ariadne TUI user guide
+
+This is a user guide for Ariadne text-based user interface (TUI), which runs in any terminal
+emulator.
+
+## About mouse support
+
+Note that almost any action accomplishable with keyboard can also be performed with a mouse in an
+intuitive way. This guide will mostly focus on keyboard shortcuts, mentioning mouse where its usage
+is not obvious.
+
+Also please note that while Ariadne has no native support for text selection with mouse (yet), you
+can still select text by using your terminal emulator's features. On Linux you usually need to hold
+`Shift` while selecting. On macOS please consult your terminal emulator's documentation.
```

Comment:
I'd write `On macOS try holding Alt or consult your terminal emulator's documentation.`, because most people use iTerm, AFAIK, and there Alt worked for me.